lib/assetmanager.js

```diff
@@ -322,6 +306,7 @@
         
         var layerIdsWithComponents = [],
             compComponents = [],
+            renderingsRequested = 0,
```

Comment:
No, I don't think it needs to be.  Its lifespan is just this `_init` method, and the purpose is simply to determine if at least one rendering is requested.